rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/CloudProviderBakeHandler.groovy

```diff
@@ -140,7 +141,21 @@ abstract class CloudProviderBakeHandler {
     def virtualizationSettings = findVirtualizationSettings(region, bakeRequest)
 
     BakeOptions.Selected selectedOptions = new BakeOptions.Selected(baseImage: findBaseImage(bakeRequest))
-    def (imageName, appVersionStr, packagesParameter) = imageNameFactory.deriveImageNameAndAppVersion(bakeRequest, selectedOptions)
+
+    List<String> packageNameList = bakeRequest?.package_name?.tokenize(" ")
```

Comment:
`bakeRequest` can't really be `null` here, so we don't need the null safe navigation between it and `package_name`.